docker-compose.yaml

```diff
@@ -1,3 +1,19 @@
 ---
 
-# This file is mandatory if your're building a new container - please remove instead it if it's not needed
+version: '2'
+
+services:
+
+  lodview:
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    container_name: pdnd-ontopia-lodview
+    image: teamdigitale/pdnd-ontopia-lodview
+    environment:
+      LODVIEW_SPARQLENDPOINT: "https://ontopia-virtuoso.dev.pdnd.italia.it/sparql"
```

Comment:
I don't like this. If really needed, I would instead download locally the image of virtuoso and run it locally. Then you put it as a dep of this container. You can take the example from ckan that does the same thing with other containers (like docker or redis)